### ISSUE TITLE
rsync hangs

### ISSUE THREAD

**Issue by sknorr** [openSUSE/docserv#218]
It seems we have rsync-related thread hangs. Apparently, rsync never finishes(?), when you \^C the program, it tells you how many rsyncs it just needed to cancel.

We do still get data on the other side though. It is weird.

**Comment by sknorr** [comment]
Okay, apparently our rsyncs are just taking way too long because:
* we are always syncing everything (we should instead sync only top-level files [.htaccess, favicon.ico], top-level lang files [index/unsupported]
* we are running multiple rsyncs that sync everything at once (#207)

**Comment by fsundermeyer** [comment]
Agreed, this is a most pressing issue. rsync only needs to sync files that have changed.

Also, the script needs to check whether an rsync process is currently running. If so, it needs to be queued.

**Comment by fsundermeyer** [comment]
We also need a cron job that regularly syncs the external tree. It also should only run if no other rsync processes are running. A simple shell script that runs every 30 minutes and aborts when `pidof rsync` retruns != 0 is probably sufficient.
